### PR TITLE
lsp-send-execute-command: better error message when fail to execute

### PR DESCRIPTION
**Pull request by kiennq**
As the title, we need better error message  in case server do not provide `workspace/executeCommand` or if there's error message when executing `lsp--send-execute-command`.

### CODE REVIEW

**Review by yyoncho — APPROVED** [review]

File: lsp-mode.el [hunk 1]
```diff
@@ -5767,7 +5767,12 @@ REFERENCES? t when METHOD returns references."
   (let ((params (if args
                     (list :command command :arguments args)
                   (list :command command))))
-    (lsp-request "workspace/executeCommand" params)))
+    (condition-case-unless-debug err
+        (lsp-request "workspace/executeCommand" params)
+      (error
+       (lsp--error "Please open an issue in lsp-mode for implementing `%s'.\n\n%S"
```
Comment (ericdallo) [inline lsp-mode.el:5773]: What if server return an error on `workspace/executeCommand`?

Comment (kiennq) [reply lsp-mode.el:5773]: We will throw other error message for that. But yes, it should be handle differently.

Comment (kiennq) [reply lsp-mode.el:5773]: On other thought, when server fail to handle `workspace/executeCommand`, we should still ask user open a bug for client handle that command.
Server may choose to only handle some command with `workspace/executeCommand`

Comment (ericdallo) [reply lsp-mode.el:5773]: I see, thanks

**Review by ericdallo — APPROVED** [review]

**Pull request merged**